### PR TITLE
implement preferences, take pictures, ask interval

### PR DESCRIPTION
**Pull request by mipmip**
Hi,
I implemented a simple preferences window, and a clean method for saving the and using the prefs. I used MotionKit, which I think is the most clean Layout Kit for RubyMotion but of course this is a matter of taste. I hope you like it.

### CODE REVIEW

File: app/app_delegate.rb [hunk 1]
```diff
@@ -46,7 +59,11 @@ def ask_and_schedule
       alert.runModal
     end
     # -5..5 + 20 yields a range of 15-25 minutes.
-    wait_time = (((rand*10).to_i-5)+20)*60
+
+    interval = NSUserDefaults.standardUserDefaults.integerForKey('AskInterval')
+    p interval
```
Comment (cyberfox) [inline app/app_delegate.rb]: Pull the debug prints out.

File: app/prefs_window/prefs_window_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class PrefsWindowController < NSWindowController
+
+  def layout
+    @layout ||= PrefsWindowLayout.new
+  end
+
+  def init
+
+    debugSettings
```
Comment (cyberfox) [inline app/prefs_window/prefs_window_controller.rb]: Same; probably shouldn't print out debugging stuff.

File: app/prefs_window/prefs_window_layout.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    editable false
+    selectable false
+    bordered false
+    bezeled false
+
+    cell do
+      alignment NSRightTextAlignment
+      scrollable false
+      drawsBackground false
+    end
```
Comment (cyberfox) [inline app/prefs_window/prefs_window_layout.rb]: You should be able to extract lines 22-31 to a 'label_style' function, and just call it from each of the labels (line 74 below).

**Review by cyberfox — COMMENTED** [review]
Maybe add an 'OK' button, or something, and/or some way of dismissing the dialog without clicking the red button.  Pressing ESC should work.

**Comment by mipmip** [comment]
Here are the changes you requested. Thanks.

**Comment by cyberfox** [comment]
👍

**Pull request merged**